src/components/r4-app.js

```diff
@@ -74,6 +74,15 @@ export default class R4App extends LitElement {
 
 		supabase.auth.onAuthStateChange(async (event, session) => {
 			if (event === 'SIGNED_OUT') this.removeDatabaseListeners()
+
+			if (event === "PASSWORD_RECOVERY") {
+				const newPassword = prompt("What would you like your new password to be?");
```

Comment:
using a prompt here, probably not the best UI, but works for now

Reply:
a prompt?! :}